modules/components/style.jsx

```diff
@@ -0,0 +1,48 @@
+var React = require('react');
+var hyphenateStyleName = require('react/lib/hyphenateStyleName');
+var forEach = require('lodash.foreach');
+var reduce = require('lodash.reduce');
+
+function buildCssString (selector, rules) {
+  // TODO: Better error checking for rules object
+  if (!selector || !rules) {
+    return;
+  }
+
+  var cssString = selector + "{";
+
+  // Turn rules into css properties
+  cssString += reduce(rules, function (s, val, styleName) {
+    return s + hyphenateStyleName(styleName) + ":" + val + ";"
+  }, "");
+
+  // Close selector
+  cssString += "}";
+
+  return cssString;
+}
+
+var Style = React.createClass({
+  render: function () {
+    var styles;
+
+    // Throw error if someone passes in valid children that isn't an object
+    if (this.props.children && typeof this.props.children !== "object") {
+      throw new Error("Style tag requires an object for 'props.children'");
+    }
+
+    forEach(this.props.children, function (rules, selector) {
+      styles = buildCssString(selector, rules);
```

Comment:
Do we want a reduce here? This replaces `styles` with each iteration, no? Doesn't support multiple rulesets.